PartymodeAutostart.py

```diff
@@ -151,7 +151,7 @@ def _viewCountdown(self):
 
             seconds = str((startDelayPartyModeSeconds - time) / 1000) + ' seconds'
 
-            conutdownDlg.update(percent, '', '', seconds)
+            conutdownDlg.update(percent)
```

Comment:
There is a reason why you have removed the seconds left?
`xbmcgui.DialogProgress().update(percent[, message])`

Reply:
kodi api changed. see https://codedocs.xyz/AlwinEsch/kodi/group__python___dialog_progress.html#gaa30bc06049eb516987569ae65f857cff

Reply:
The method signature is changed, but we can always print the reaming seconds :)